components/DevTimeStamp/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import dynamic from 'next/dynamic'
+import './DevTimeStamp.module.css'
+import {
+  Box,
+  Flex,
+  Text,
+} from "@chakra-ui/react";
+
+const getTimeStampFormatted = (rawTimestamp: number) => {
```

Comment:
Maybe `dayjs` would be a good fit here since we don't have to deal with digits, formatting, or casting to string manually, etc. Working with dates might be a headache and might require many workarounds.

If this solves the problem, lgtm!

Reply:
I thought about that but figured bringing in a package for a single instance of date formatting is overkill

Reply:
Yeah, extra dependency is not good. Since it's quite small lib and provides tree shaking approach for other modules, base package is tiny and still worth using it for anything related to dates imo